### PR TITLE
add new hosted CE to Clemson

### PR DESCRIPTION
**Pull request by coldbeer101**
Adding new hosted CE to Clemson Palmetto. Need an id still

### CODE REVIEW

**Comment by coldbeer101** [comment]
Ooops I merged FIU into my master and did pull request. Forgot the Clemson PR was still waiting. FIU Needs SITE id and resource ID

File: topology/Clemson University/Clemson-Palmetto/Clemson-Palmetto.yaml [hunk 1]
```diff
@@ -66,4 +66,86 @@ Resources:
...
       StorageCapacityMin: 0
+  Clemson-Palmetto-CE1:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: true
+    # Description is a long description of the resource; may be multiple lines
+    Description: Hosted CE for Clemson-Palmetto
+    # If you have an up-to-date local git clone, fill ID with the output from `bin/next_resource_id`
+    # Otherwise, leave it blank and we will fill in the appropriate value for you.
+    ID:
```
Comment (edquist) [inline topology/Clemson University/Clemson-Palmetto/Clemson-Palmetto.yaml:77]:
```suggestion
    ID: 1413
```

File: topology/Florida International University/FIU-IRCC/FIU-IRCC.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+GroupDescription: Florida International University Instructional Research and Computing Center
+GroupID:
+Production: true
+Resources:
+  FIU-IRCC-CE1:
+    Active: true
+    Description: This is a Hosted CE for FIU IRCC group
+    ID:
```
Comment (edquist) [inline topology/Florida International University/FIU-IRCC/FIU-IRCC.yaml:8]:
```suggestion
    ID: 1414
```

File: topology/Florida International University/FIU-IRCC/SITE.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+LongName: FIU HPC Division of Information Technology
+Description: FIU Panther cluster sponsored by campus central IT division.
+AddressLine1: 11200 SW 8th St
+AddressLine2: PC415b
+City: Miami
+Country: United States
+ID:
```
Comment (edquist) [inline topology/Florida International University/FIU-IRCC/SITE.yaml:7]:
```suggestion
ID: 10355
```

**Review by edquist — COMMENTED** [review]
See suggestions to add missing site/resource IDs

File: topology/Florida International University/FIU-IRCC/FIU-IRCC.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+GroupDescription: Florida International University Instructional Research and Computing Center
+GroupID:
```
Comment (edquist) [inline topology/Florida International University/FIU-IRCC/FIU-IRCC.yaml:2]:
```suggestion
GroupID: 1342
```

File: topology/Florida International University/FIU-IRCC/FIU-IRCC.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+          ID: 3ef2e11c271234a34f154e75b28d3b4554bb8f63
+        Secondary:
+          Name: Jeffrey Michael Dost
+          ID: 3a8eb6436a8b78ca50f7e93bb2a4d1f0141212ba
+
+    FQDN: fiu-ircc-ce1.svc.opensciencegrid.org
+
+    Services:
+      CE:
+        Description: Hosted CE for FIU-IRCC
```
Comment (edquist) [inline topology/Florida International University/FIU-IRCC/FIU-IRCC.yaml:30]:
```suggestion
        Description: Hosted CE for FIU-IRCC
        
SupportCenter: Self Supported
```

**Review by edquist — COMMENTED** [review]
Additional missing GroupID and SupportCenter

**Review by edquist — APPROVED** [review]

**Pull request merged**